plone-5/instance/src/ukstats.ccv2.content_types/src/ukstats/ccv2/content_types/api/controlpanel.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.restapi.services import Service
+from plone.restapi.serializer.converters import json_compatible
+
+reg_base = 'cmsconf'
+
+
+class SiteTitle(Service):
+    def reply(self):
+        registry_record_value = api.portal.get_registry_record(reg_base + '.site_title')
+        response_data = {
+            'site_title': registry_record_value
+        }
+        return json_compatible(response_data)
+
+
+class PhaseBanner(Service):
+    def reply(self):
+        phasebanner_state = api.portal.get_registry_record(
+            reg_base + '.phasebanner_state')
+        feedback_email = api.portal.get_registry_record(reg_base + '.feedback_email')
+        return {'bannerDisplay': phasebanner_state, 'bannerLink': feedback_email}
```

Comment:
should this have a `json_compatible` around the return? Not quite sure what that does but one out of four seems odd.